File: BackEnd/BackEnd/Models/Identificacion.cs
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.Models
{
    public class Identificacion
    {
        [BsonElement("numero")]
        public string Numero { get; set; }

        [BsonElement("fecha_expedicion")]
        public DateTime Fecha_Expedicion { get; set; }


        [BsonElement("lugar_expedicion")]
        public DateTime Lugar_Expedicion { get; set; }


        [BsonElement("tipo")]
        public string Tipo { get; set; }
    }
}

File: FrontEnd/FrontEnd/src/app/models/identificacion.ts
export class Identificacion{
  numero: string;
  fecha_expedicion: string;
  lugar_expedicion: string;
  tipo: string;


  constructor(
    numero: string,
    fecha_expedicion: string,
    lugar_expedicion: string,
    tipo: string
) {
    this.numero = numero
    this.fecha_expedicion = fecha_expedicion
    this.lugar_expedicion = lugar_expedicion
    this.tipo = tipo
  }

}
